content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -76,7 +81,10 @@ Then, if working in a multi-company environment, select which company this price
 in the :guilabel:`Company` field. If this field is left blank, the pricelist is automatically
 applied to all companies in the database.
 
-Price rules tab
+It should also be noted that, in Odoo 17 (and above), a chatter section can be found on pricelist
```

Comment:
would also add this to the **Note** block since it's similar version-specific content, and so it becomes easy to edit/remove later